x-pack/plugins/ingest_manager/common/constants/routes.ts

```diff
@@ -14,10 +14,12 @@ export const FLEET_API_ROOT = `${API_ROOT}/fleet`;
 export const LIMITED_CONCURRENCY_ROUTE_TAG = 'ingest:limited-concurrency';
 
 // EPM API routes
+const EPM_PACKAGES_BULK = `${EPM_API_ROOT}/packages_bulk`;
```

Comment:
`packages/_bulk` like ES also occurred to me but I haven't looked into it.

Reply:
I just put up a bulk actions PR https://github.com/elastic/kibana/pull/77690 where I added `/bulk_reassign` and `/bulk_unenroll` routes. does it make sense to agree on a convention here?

Reply:
I was gonna ask the same thing. Seeing yours makes me say keep bulk_*.

Reply:
Thanks for the feedback so are we thinking `bulk_packages` then? Or should I go with `packages/_bulk` if that works?

Reply:
since we don't have a verb when installing packages in the route, seems like `packages/_bulk` would make sense.

Reply:
good point, @neptunian . `packages/_bulk` was my first instinct, but I don't have a strong opinion at the moment.

Reply:
imo this points to the need to add a verb to this route. is my understanding correct that this route is only for _upgrading_ existing packages, rather than being able to bulk _install_ packages? if so, I think it makes sense to differentiate this from the install routes (`/packages`). what do you all think of using `/packages/bulk_upgrade`?:
```
BULK_UPGRADE_PATTERN: `${EPM_PACKAGES_MANY}/bulk_upgrade`
```

Reply:
@jen-huang It can do both upgrades and installs. If the package doesn't exist, it'll just install the latest one available.

Reply:
in that case I'm happy to leave the route as `/packages_bulk`, or simply switching to `bulk_packages` to follow more closely with the bulk actions API convention. although `/_bulk` is an ES convention, I've never felt that Kibana APIs need to follow ES patterns

Reply:
@jen-huang @neptunian @jfsiii I'm happy to switch it to `bulk_packages` unless someone objects?

Reply:
I'm not too fussed but I don't think `bulk_packages` really follows the convention because those are using `bulk_(verb)` like `bulk_upgrade`.  Since we don't have the verb, it seems redundant to say `packages/bulk_packages`.